examples/send-receive/Cargo.toml

```diff
@@ -10,14 +10,14 @@ version = "0.1.0"
 
 [features]
 default = ["ubx_proto14", "ubx_proto23", "ubx_proto27", "ubx_proto31"]
-ubx_proto14 = ["ublox_device/ubx_proto14"]
-ubx_proto23 = ["ublox_device/ubx_proto23"]
-ubx_proto27 = ["ublox_device/ubx_proto27"]
-ubx_proto31 = ["ublox_device/ubx_proto31"]
+ubx_proto14 = ["ublox-device/ubx_proto14"]
+ubx_proto23 = ["ublox-device/ubx_proto23"]
+ubx_proto27 = ["ublox-device/ubx_proto27"]
+ubx_proto31 = ["ublox-device/ubx_proto31"]
 
 [dependencies]
 chrono = "0.4"
 clap = { version = "4.2", features = ["cargo"] }
-serialport = "4.2"
+serialport = { version = "4.2", default-features = false }
```

Comment:
Was this the leading cause of the `libudev` in CI?

Reply:
yes